docs/app/Examples/modules/Popup/Types/PopupHtmlExample.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Popup, Card, Rating, Image } from 'stardust'
+
+const users = [
```

Comment:
And you are right, I'm so confused :disappointed: It was a left over from a copy/paste and indeed my linter catch it too. Trying to do too many things at the same time.